dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/DotAssetViewStrategy.java

```diff
@@ -0,0 +1,74 @@
+package com.dotmarketing.portlets.contentlet.transform.strategy;
+
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformOptions.USE_ALIAS;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.FILE_NAME_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.MIMETYPE_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.TITLE_FIELD;
+import static com.dotmarketing.util.UtilHTML.getIconClass;
+import static com.dotmarketing.util.UtilHTML.getStatusIcons;
+import static com.dotmarketing.util.UtilMethods.getFileExtension;
+
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.model.ResourceLink;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+import java.io.File;
+import java.util.Map;
+import java.util.Set;
+
+public class DotAssetViewStrategy extends WebAssetStrategy<Contentlet> {
+
+    private static final String ASSET = "asset";
+
+    DotAssetViewStrategy(final TransformToolbox toolBox) {
+        super(toolBox);
+    }
+
+    @Override
+    public Contentlet fromContentlet(final Contentlet contentlet) {
+        return contentlet;
+    }
+
+    @Override
+    public Map<String, Object> transform(final Contentlet dotAsset, final Map<String, Object> map,
+            final Set<TransformOptions> options, User user)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'user' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)